settings/authentication-personalization/authentication-setup/oauth.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title: 'OAuth 2.0 Handshake'
+description: 'Integrate with your OAuth server to enable user login via the PKCE flow'
```

Comment:
is this still accurate even tho we got rid of PKCE?

Reply:
Noooo good catch

Reply:
fixed